Kickstarter-iOS/Views/Cells/PledgeShippingLocationCell.swift

```diff
@@ -107,3 +116,22 @@ private let countryButtonTitleLabelStyle: LabelStyle = { (label: UILabel) in
   label
     |> \.lineBreakMode .~ .byTruncatingTail
 }
+
+// MARK: - Functions
+
```

Comment:
Add `//TODO:` here to move this function to the future `PledgeShippingLocationCellViewModel`